From aa9abe7692a3ee99b69811594938d97cb180351e Mon Sep 17 00:00:00 2001
From: avi <avi@chromium.org>
Date: Wed, 20 May 2015 13:57:17 -0700
Subject: [PATCH] Use the correct URL for ERR_SSL_WEAK_SERVER_EPHEMERAL_DH_KEY
 errors.

BUG=490260,490240
TEST=as in bug

Review URL: https://codereview.chromium.org/1148943002

Cr-Commit-Position: refs/heads/master@{#330799}
---
 chrome/common/localized_error.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/common/localized_error.cc b/chrome/common/localized_error.cc
index 0c4ad43..1ea56b1 100644
--- a/chrome/common/localized_error.cc
+++ b/chrome/common/localized_error.cc
@@ -49,7 +49,7 @@ namespace {
 static const char kRedirectLoopLearnMoreUrl[] =
     "https://www.google.com/support/chrome/bin/answer.py?answer=95626";
 static const char kWeakDHKeyLearnMoreUrl[] =
-    "http://sites.google.com/a/chromium.org/dev/"
+    "https://www.chromium.org/administrators/"
     "err_ssl_weak_server_ephemeral_dh_key";
 #if defined(OS_CHROMEOS)
 static const char kAppWarningLearnMoreUrl[] =
-- 
2.4.2

